qemu/deps/win_driver_install/win_driver_install.py

```diff
@@ -202,7 +202,7 @@ def show_log_output(result_file):
     result_file = r"C:\driver_install.log"
     logger = logging.getLogger("driver_install")
     logger.setLevel(logging.DEBUG)
-    fh = logging.FileHandler(result_file)
+    fh = logging.FileHandler(result_file, mode="a+")

```

Comment:
There are 3 different cases.
1-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.single_driver_install.with_netkvm
2-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.single_driver_install.with_viorng
3-smp_8.8192m.repeat1.run_test.Host_RHEL.m7.u3.qcow2.virtio_blk.up.virtio_net.Win2012.x86_64.r2.io-github-autotest-qemu.single_driver_install.with_viostor

2016-07-27 16:05:09,387 - driver_install - DEBUG - dpinst.exe is already existed
2016-07-27 16:05:09,464 - driver_install - INFO - Install certificate!
2016-07-27 16:05:09,464 - driver_install - DEBUG - Sending command: certutil -addstore -f TrustedPublisher D:\redhat.cer
2016-07-27 16:05:10,434 - driver_install - DEBUG - Command output is []
2016-07-27 16:05:10,434 - driver_install - INFO - Install driver netkvm!
2016-07-27 16:05:10,434 - driver_install - DEBUG - Sending command: C:\dpinst.exe /A /PATH E:\netkvm\2k12R2\amd64 /C /LM /Q /F
2016-07-27 16:05:12,903 - driver_install - DEBUG - Command output is []
2016-07-27 16:06:17,095 - driver_install - INFO - Compare whether driver version is same as expected.
2016-07-27 16:06:17,111 - driver_install - INFO - Current driver version 62.72.104.11000 is same as expected.
2016-07-27 16:08:17,730 - driver_install - DEBUG - dpinst.exe is already existed
2016-07-27 16:08:17,746 - driver_install - DEBUG - dpinst.xml is already existed
2016-07-27 16:08:17,746 - driver_install - INFO - Install certificate!
2016-07-27 16:08:17,746 - driver_install - DEBUG - Sending command: certutil -addstore -f TrustedPublisher D:\redhat.cer
2016-07-27 16:08:18,839 - driver_install - DEBUG - Command output is []
2016-07-27 16:08:18,839 - driver_install - INFO - Install driver viorng!
2016-07-27 16:08:18,839 - driver_install - DEBUG - Sending command: C:\dpinst.exe /A /PATH E:\viorng\2k12R2\amd64 /C /LM /Q /F
2016-07-27 16:08:22,917 - driver_install - DEBUG - Command output is []
2016-07-27 16:09:26,638 - driver_install - INFO - Compare whether driver version is same as expected.
2016-07-27 16:09:26,670 - driver_install - INFO - Current driver version 62.72.104.10500 is same as expected.
2016-07-27 16:11:48,641 - driver_install - DEBUG - dpinst.exe is already existed
2016-07-27 16:11:48,673 - driver_install - DEBUG - dpinst.xml is already existed
2016-07-27 16:11:48,688 - driver_install - INFO - Install certificate!
2016-07-27 16:11:48,688 - driver_install - DEBUG - Sending command: certutil -addstore -f TrustedPublisher E:\redhat.cer
2016-07-27 16:11:49,736 - driver_install - DEBUG - Command output is []
2016-07-27 16:11:49,736 - driver_install - INFO - Install driver viostor!
2016-07-27 16:11:49,736 - driver_install - DEBUG - Sending command: C:\dpinst.exe /A /PATH F:\viostor\2k12R2\amd64 /C /LM /Q /F
2016-07-27 16:11:52,438 - driver_install - DEBUG - Command output is []
2016-07-27 16:13:03,483 - driver_install - INFO - Compare whether driver version is same as expected.
2016-07-27 16:13:03,499 - driver_install - INFO - Current driver version 62.72.104.10600 is same as expected.